.github/workflows/reassurePerformanceTests.yml

```diff
@@ -24,22 +22,6 @@ jobs:
             git config --global user.email "test@test.com"
             git config --global user.name "Test"
 
-      - name: Get common ancestor commit
-        run: |
-            git fetch origin main
-            common_ancestor=$(git merge-base "${{ github.sha }}" origin/main)
-            echo "COMMIT_HASH=$common_ancestor" >> "$GITHUB_ENV"
-
-      - name: Clean up deleted files
-        run: |
-            DELETED_FILES=$(git diff --name-only --diff-filter=D "$COMMIT_HASH" "${{ github.sha }}")
-            for file in $DELETED_FILES; do
-              if [ -n "$file" ]; then
-                rm -f "$file"
-                echo "Deleted file: $file"
-              fi
-            done
```

Comment:
Why did we need this deletion logic before but don't need it now?

Reply:
This logic was originally added to resolve an issue where files weren’t properly deleted on CI, causing Reassure tests to fail by detecting duplicate files with the same tests. The issue occurred during the migration to TypeScript when test file names were changed. Now that the migration is complete, we no longer need this logic.

Reply:
I think we still need this. We are facing an issue where after a patch file is renamed the instance sees two patches and patch-package fail https://github.com/Expensify/App/actions/runs/12285318556/job/34283030276?pr=53627